src/quick-start.md

```diff
@@ -58,23 +72,23 @@ spec:
 
 Overview of the attributes of the `PolicyServer` resource:
 
-* `image`: container image name
-* `replicas`: number of desired instances
-* `serviceAccountName` (optional): `policy-server` ServiceAccount. Namespace's `default` ServiceAccount will be used if nothing is provided
-* `env` (optional): `policy-server` environment variables
-* `annotations` (optional): `policy-server` annotations
+| Required | Placeholder         | Description    |
+|:--------:| ------------------- | ----------------------------- |
+| [x] | `image`  | The name of the container image |
```

Comment:
Nit: tried building the book locally (from the .github/workflow, it seems to be `mdbook build`) and the `[x]` don't render on the html that would get deployed.

Reply:
Good catch, maybe we can replace that with a emojii like :heavy_check_mark: